specifyweb/frontend/js_src/lib/components/Toolbar/DataModel.tsx

```diff
@@ -310,6 +348,39 @@ function DataModelRelationships({
   );
 }
 
+function DependentFilterButton({
+  isPressed,
+  children,
+  onClick: handleClick,
+  className: newClasses,
+}: {
+  readonly isPressed: boolean;
+  readonly children: LocalizedString;
+  readonly onClick: () => void;
+  readonly className?: string;
+}): JSX.Element {
+  return (
+    <button
```

Comment:
Not good.
Don't use `<button>` - use `<Button.SomeButtonVariantHere>` instead - it comes with a bunch of styles baked in so you don't have to write the same styles every time.
With your solution, imagine if I need to change the border of the button in dark mode - now I have to look for every simple possible way where you may have used this button.

Even better - there already is a <RadioButton> component in `./components/AppResources/Filters.tsx`- you can just export it and reuse it here rather than have to duplicate the code